bin/MindeeCLICommand.php

```diff
@@ -0,0 +1,336 @@
+    {
+        $this->addOption(
+            'async_polling',
+            'A',
+            InputOption::VALUE_NONE,
+            'When enabled, enqueues and parses the document asynchronously.'
+        )
+            ->addOption(
+                'pages_remove',
+                'r',
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+                'Indexes of the pages to remove from the document.'
+            )
+            ->addOption(
+                'pages_keep',
+                'p',
+                InputOption::VALUE_REQUIRED,
+                'Indexes of the pages to keep in the document.'
+            )
+            ->addOption(
+                'key',
+                'k',
+                InputOption::VALUE_OPTIONAL,
+                'API key for the account. Is retrieved from environment if not provided.'
+            )
+            ->addOption(
+                'output_type',
+                'o',
+                InputOption::VALUE_REQUIRED,
+                'Specify how to output the data.
+ - summary: a basic summary (default)
+ - raw: the raw HTTP response
+ - parsed: the validated and parsed data fields
+'
+            )
+            ->addOption(
+                'full_text',
+                't',
+                InputOption::VALUE_NONE,
+                "Include full document text in response."
+            )
+            ->addOption(
+                'cropper',
+                'c',
+                InputOption::VALUE_NONE,
+                "Apply cropper operation to the document (if available)."
+            )
+            ->addOption(
+                'debug',
+                'D',
+                InputOption::VALUE_NONE,
+                'Debug mode (dry-run).'
+            );
+    }
+
+    private function configureCustomOptions()
+    {
+        $this
+            ->addOption(
+                'account_name',
+                'a',
+                InputOption::VALUE_REQUIRED,
+                'API account name for the endpoint'
+            )
+            ->addOption(
+                'endpoint_name',
+                'e',
+                InputOption::VALUE_REQUIRED,
+                'API endpoint name for the endpoint'
+            )
+            ->addOption(
+                'endpoint_version',
+                'd',
+                InputOption::VALUE_OPTIONAL,
+                'Version for the endpoint. If not set, use the latest version of the model'
+            );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $args = $input->getArguments();
+        $opts = $input->getOptions();
+
+        if (count(array_filter($args)) <= 0 && count(array_filter($opts)) <= 0) {
+            $output->writeln($this->formatHelp(), OutputInterface::OUTPUT_NORMAL);
+            exit(Command::FAILURE);
+        }
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Without getting all Rubocop on you, this method should be split into smaller ones.

This first method could do all the checks for example, if no errors then call a second method.